test/accountSpec.js

```diff
@@ -11,10 +11,32 @@ const dbHelper = require('./helpers/db')
 const appHelper = require('./helpers/app')
 const logHelper = require('five-bells-shared/testHelpers/log')
 
+const accountValidator = require('../src/services/validator').create('Account')
+const transferValidator = require('../src/services/validator').create('Transfer')
 const publicKey = fs.readFileSync(__dirname + '/data/public.pem', 'utf8')
 
 const START_DATE = 1434412800000 // June 16, 2015 00:00:00 GMT
 
+function validAccounts (res) {
+  res.body.forEach(account => {
+    validAccount({body: account})
+  })
+}
+
+function validAccount (res) {
+  const validatorResult = accountValidator(res.body)
+  if (!validatorResult.valid) {
+    throw new Error('Account schema validation error: ' + JSON.stringify(_.omit(validatorResult.errors[0], ['stack'])))
+  }
+}
+
+function validTransfer (res) {
```

Comment:
Let's discuss the naming convention here. I agree that, as a function, this should most likely be called `validateTransfer`, but with the BDD chai `expect`, it kind of makes sense to call it `validTransfer` (we expect a valid transfer to be returned from the endpoint). I'm open to suggestions.